--- ini/jk_init.ini.orig	2017-05-30 01:04:49.000000000 -0500
+++ ini/jk_init.ini	2017-05-30 01:10:07.000000000 -0500
@@ -151,3 +151,7 @@
 #comment = xterm
 #paths = /usr/bin/X11/xterm, /usr/share/terminfo, /etc/terminfo
 #devices = /dev/pts/0, /dev/pts/1, /dev/pts/2, /dev/pts/3, /dev/pts/4, /dev/ptyb4, /dev/ptya4, /dev/tty, /dev/tty0, /dev/tty4
+
+[php]
+comment = modules and config files for PHP
+paths = /usr/bin/php /etc/php.ini /etc/php.d /usr/lib64/php
